exploration/delimiting-variant-patterns.md

```diff
@@ -175,13 +221,16 @@ Pros:
 - The quoting reduces the number of in-pattern escapes to the open/close sequence.
   and the placeholder sequence sigils.
 - Since the pattern is already quoted, translators never have to add pattern quotes
-  in order to add PEWs to a given pattern.
+  in order to include whitespace into a given pattern.
```

Comment:
```suggestion
  in order to add leading or trailing whitespace in a given pattern.
```

Reply:
Mostly good, but "include"/"into" I think is slightly clearer.

```suggestion
in order to include leading or trailing whitespace into a given pattern.
```